gaps/gap-1_gap_process/gap-1_gap_process.md

```diff
@@ -0,0 +1,44 @@
+---
+gap: 1
+title: GAP Process
+description: Specification of Golem Amendment Proposal process.
+author: stranger80 (@stranger80)>
+status: Draft
+type: Meta
+---
+
+## Abstract
+We are introducing a Golem Amendment Proposal process to establish governance over changes proposed, considered and implemented in Golem ecosystem.
+
+## Motivation
+The GAP process responds to a number of needs:
+- Establish governance over features, specification/standards amendments and process improvements managed by Golem Factory.
+- Ensure visibility of features in the Golem pipeline - this is especially valuable for interested parties outside of Golem Factory.
+- Provide ability for Golem community to submit proposals for features in a controlled, systematic manner.
```

Comment:
```suggestion
- Open a path to enable the Golem community to submit proposals for features in a controlled, systematic manner.
```
We need to try this out by ourselves in-house for some time.

Reply:
Fair enough.